docs/en/prerequisites.rst

```diff
@@ -1,6 +1,49 @@
 Prerequisites
 ===============================
-The minimum requirements for running the ESP-IDF Installation Manager are below.
+Below are the minimum requirements for running the ESP-IDF Installation Manager.
+
+Windows
+----------------
+
+- git
+- cmake
+- ninja
+- python with pip capable of creating virtual environment and doing SSL requests
+
```

Comment:
For Windows, Git and CMake are not prerequisites for installing eim, right? They are prerequisites for IDF, they were installed by eim.

Reply:
the prerequisites are always for the IDF as eim does not have any and as it's stated in the note, on windows we can take care of these if user wants to